test/e2e_node/eviction_test.go

```diff
@@ -473,13 +473,13 @@ var _ = SIGDescribe("PriorityPidEvictionOrdering", framework.WithSlow(), framewo
 
 	highPriorityClassName := f.BaseName + "-high-priority"
 	highPriority := int32(999999999)
-	processes := 30000
+	processes := 5000
```

Comment:
thanks for the PR! I agree we should have this fix to restore some signal.

I would prefer to add a comment here (and to refer on line 482 below) about how you determined these numbers, which can probably be just a copypaste of your previous github comments.

The reason I'm asking is that keeping history is hard, keeping history from github issue/PR comments is extra hard.